### ISSUE TITLE
AHDM PDF queries, issues, suggested corrections

### ISSUE THREAD

**Issue by JackWilton1594** [projectLEMDO/lemdoIssues#249]
Hello folks!

I'm working through the AHDM PDF, and am very pleased with how slick it's all looking – very cool!

I have compiled a list of queries, issues, and suggested corrections – see below.

I'll now turn my attention to the text and annotations. :)

Thanks!
Brett

- - - - -

**Title Page**
1. Can we italicise the play title?

**Table of Contents**
1. Can we italicise the play title?
2. Can we remove the play title preceding "Acknowledgements"?

**General Introduction**
1. Can we use bold weight for the headings?
2. Can we remove the play title from the running headers? (e.g. _An Humorous Day's Mirth:_ George Chapman (1559–1634): "The best for Comedy" >> George Chapman (1559–1634): "The best for Comedy")?
3. Can we remove double spaces after periods? These are not present in the XML, and produces especially inelegant rendering when lines are cited parenthetically using "l." and "ll." (Unless these are an unavoidable product of the text justification?)
4. Figure 7 is a duplication of Figure 4 and should be removed. (This will necessitate renumbering the subsequent figures.)
5. Technically, none of the figures are under copyright. All instances of "© X" in the figure captions should become "Courtesy X".
6. Rather than "ScX SpY" for internal citations, can we not use "X.SpY"? I'm just thinking ahead to other editions where they use acts as well as scenes, and "AcX ScY SpZ" looks especially clunky (as opposed to "X.Y.SpZ")

**Modern Text** 
1. Running header: can we expand the reference appearing after the play title to include the appropriate act/scene? For example, rather than "_An Humorous Day's Mirth_: 2", we'd have "_An Humorous Day's Mirth_: Scene 2", and so on.
2. Annotations to the scene division should be footnoted on the same line – i.e., the footnote should not be rendered on the following line with the opening stage direction. For example: 
<img width="645" alt="Screenshot 2024-07-25 at 15 30 49" src="https://github.com/user-attachments/assets/802b20cb-9b8d-4497-8a6f-627eea86a04d">
3. Formatting of internal citations (as per note 6 above): Rather than "ScX SpY" for internal citations, can we not use "X.SpY" and "X.Y.SpZ" for "AcX ScY SpZ"?
4. Same issue with double spaces as noted above.
5. Can we use bold weight for the act/scene divisions?
6. Is it possible to avoid beginning scenes at the end of pages (see scenes 6, 11, and 12)? For example:
<img width="714" alt="Screenshot 2024-07-25 at 15 41 44" src="https://github.com/user-attachments/assets/acb72dc5-89c8-4964-9ca3-a42d3f9e5f8b">

**Bibliography**
1. Same issue with double spaces following some periods as noted above.

**Acknowledgements**
1. Please remove "_An Humorous Day's Mirth_:" from section title.
2. Same issue with double spaces following some periods as noted above.

**Comment by JackWilton1594** [comment]
Addenda:

**Modern Text**
1. Some entrance SDD (especially opening SDD) are indented whereas others are flush left. They should all be flush left to align with the SPP. For example:
![Screenshot 2024-07-25 at 16 28 32](https://github.com/user-attachments/assets/f06c5507-cd0c-4c09-942e-d4341dc27d87)
2. This foreign prose content is rendered with an extraneous period in the online version:
![Screenshot 2024-07-25 at 16 21 19](https://github.com/user-attachments/assets/b7a2785b-8646-4426-981e-187900d548fc)
... and without break with the verse line that follows in the print:
![Screenshot 2024-07-25 at 16 22 25](https://github.com/user-attachments/assets/f0a99099-2e1c-45b6-880b-1c5974f2ae2e)
3. Another instance of an extraneous period following foreign prose:
![Screenshot 2024-07-25 at 16 32 56](https://github.com/user-attachments/assets/2b447214-6249-4856-80d2-e00f6ed03d1f)
4. Similar problem with a prose passage followed by lines of verse without any break (verse begins "Choke"):
![Screenshot 2024-07-25 at 16 34 37](https://github.com/user-attachments/assets/25c5b59a-c37c-4bcc-a073-05665b296bd5)

**Comment by martindholmes** [comment]
@JackWilton1594 This is about 30 different tickets, and we need to separate them out. As far as I can see, they fall into at least two different categories:

1. Bug reports (such as the suggestion that there are double spaces after periods -- I don't believe we're inserting any extraneous periods, but the LaTeX justification may be giving that impression.
2. Feature requests and proposals to change rendering features we've previously agreed upon, such as removing the play title from running headers. Since these were implemented based on consensus, I think these need discussion by everyone concerned before I'm tasked with making any changes.

I'm going to make a start on creating separate issues for each case, and I'll create a new issue category for "Needs consensus" for the latter, and assign them to you and Janelle to discuss with the other editors before they get assigned to me.

**Comment by martindholmes** [comment]
It looks like the second batch of stuff relates to the website rendering, not the PDF.

**Comment by martindholmes** [comment]
Multiple periods after foreign elements may be caused by the periods following the foreign text being placed not only after the footnote anchor but also outside the foreign element itself, as though the final period of the Latin text is actually in English, which doesn't make sense. I've moved those periods, and we'll see if that solves the problem.

**Comment by JanelleJenstad** [comment]
@JackWilton1594: I've removed the duplicate image. The images will be automatically renumbered in the processing. 

@JackWilton1594: I've also removed the copyright symbol and reworded as necessary.

**Comment by martindholmes** [comment]
I've generated distinct tickets for all the issues raised here, so I'm going to close this ticket to avoid confusion.
